03/app.js

```diff
@@ -42,12 +42,34 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+    if (typeof this[isbn] === "undefined") {
+        return;
+    }
+    const title = this[isbn]["title"][lang];
+    if (title) {
+        return title;
+    }
+    return null;
 
 }
 
 books.getTranslator = function(isbn, lang) {
+    if (typeof this[isbn] === "undefined") {
+        return null;
+    }
+    if (typeof this[isbn]["translator"] === "undefined") {
+        return null;
+    }
```

Comment:
Skoro tutaj sprawdzasz też czy występuje "tłumacz" to również między linią 47 i 48 też powinieneś sprawdzić czy występuje "tytuł" :)

PS. Można to też zapisać tak: 

```
if (typeof this[isbn] === "undefined" || typeof this[isbn]["translator"] === "undefined") {
    return null;
}
```